src/ngraph/frontend/onnx_import/op/leaky_relu.cpp

```diff
@@ -42,8 +48,8 @@ namespace ngraph
                     std::shared_ptr<ngraph::Node> alpha_node =
                         std::make_shared<ngraph::op::Constant>(
                             data->get_element_type(), Shape{}, std::vector<double>{alpha});
-
-                    return {std::make_shared<ngraph::op::LeakyRelu>(data, alpha_node)};
+                    alpha_node = ngraph::op::make_broadcast_node(alpha_node, data->get_shape());
```

Comment:
Why not to replace it with PReLU op?

Reply:
We already have a PReLU op:
https://github.com/NervanaSystems/ngraph/blob/b69d3fd68e6a71baaefaab9e96302915f76f7f52/src/ngraph/op/fused/prelu.hpp#L31